src/Workspaces/Core/Portable/Remote/WellKnownRemoteHostServices.cs

```diff
@@ -6,7 +6,7 @@ internal class WellKnownRemoteHostServices
     {
         public const string RemoteHostService = "remoteHostService";
         public const string RemoteHostService_Connect = "Connect";
-        public const string RemoteHostService_SynchronizeAsync = "SynchronizeAsync";
```

Comment:
can we not use nameof() here to reduce hte number of string cases we need to handle?